src/app/components/progress-indicator/passive-progress-indicator/index.html

```diff
@@ -0,0 +1,276 @@
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don’t use passive progress indicators to represent independent steps or statuses that are not sequential.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-3.png" />
+              <div class="caption caption-danger sky-text-danger">Don't use passive progress to track steps that are not sequential.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don't use passive progress indicators simultaneously for multiple items in lists.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-separate">
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-4.png" />
+              <div class="caption caption-danger sky-text-danger">Don't display multiple passive progress indicators in lists.</div>
+            </div>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-5.png" />
+              <div class="caption caption-success sky-text-success">Do roll up steps into a single status in lists. Use a container such as <a routerLink="/components/flyout">flyouts</a> or <a routerLink="/components/popover">popovers</a> to display details from passive progress indicators.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Anatomy">
+        <sky-docs-anatomy imagePath="~/assets/img/guidelines/progress-indicator/passive-progress-anatomy.png">
+          <sky-docs-anatomy-item anatomyNumber="1">Step indicator</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="2">Step label</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item isOptional="true" anatomyNumber="3">Step details</sky-docs-anatomy-item>
+        </sky-docs-anatomy>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Options">
+        <h3>Step details</h3>
+        <div class="sky-text-block">
+          <p>You can provide additional details about completed or current steps. Use step details to provide extra context to users, but avoid too many details or the implication that users should take action.</p>
+        </div>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Content">
```

Comment:
We call out to use past tense verbs for step labels, but in the examples we only do this for completed steps. For pending steps in the examples, we generally use present tense verbs (mostly gerunds, but not always). The Content section also calls out info about datestamp and timestamp without specifying that it is for completed steps. I’ve updated the info to call out that the guidance is for completed steps. Do we need to provide additional guidance for pending steps? Also, should we provide any guidance on the cases when not to include datestamps? Or on the cases when to include timestamps? I'm not sure that "in most cases" and "if necessary" are really all that clear. (We may not be able to provide reasons examples for when to deviate from the norm here, but if not, we could still possibly make the rationale more clear. For example we could change the date stamp info to "and include datestamps in parentheses unless you have a compelling reason not to." I'm not sure if that's any better though, so just throwing this out there for your consideration.)

Reply:
![image](https://user-images.githubusercontent.com/15896944/68946849-f45afa00-0781-11ea-9969-cd2964cdb9c1.png)